NitroxModel/IPatch.cs

```diff
@@ -0,0 +1,8 @@
+namespace NitroxModel;
+
+/// <summary>
+///     Interface to mark a type as a patcher, which inject code.
+/// </summary>
+public interface IPatch
```

Comment:
Why not just `INitroxPatch`?

Reply:
`INitroxPatch` uses Harmony reference. I need it in NitroxModel which, rightfully, does not have `Harmony`.